Neuron/Sections/CKB/CKBViewController.swift

```diff
@@ -7,7 +7,29 @@
 //
 
 import UIKit
+import CKB
 
 /// CKB Support PoC.
 class CKBViewController: UIViewController {
+    private let rpcClient = APIClient(url: URL(string: "http://localhost:8114")!)
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        loadHeight()
+    }
+}
+
+extension CKBViewController {
+    private func loadHeight() {
+        rpcClient.request(method: "get_tip_header") { (json, error) in
+            if let error = error {
+                print(error)
```

Comment:
`print` just for test ?

Reply:
Yes, CKB related features are all temporary so we'll allow `print`.